mm2src/coins/utxo/wallet_connect.rs

```diff
@@ -1,19 +1,22 @@
 //! This module provides functionality to interact with WalletConnect for UTXO-based coins.
-use std::convert::TryFrom;
+use std::{collections::HashMap, convert::TryFrom};
 
+use crate::UtxoTx;
+use base64::engine::general_purpose::STANDARD as BASE64_ENGINE;
+use base64::Engine;
+use bitcoin::{consensus::Decodable, consensus::Encodable, psbt::Psbt, EcdsaSighashType};
```

Comment:
Will this use of rust bitcoin work for all UTXO coins? I know that it will work for the currently supported ones in Wallet Connect which is why we will not need to change this for now. I am just curious